.eslintrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": ["standard", "prettier"],
+  "parserOptions": {
+    "ecmaVersion": 13,
```

Comment:
I'm getting a parsing error on the specified ecmaVersion. I'm not sure if `13` is supported by eslint 7.x, I think `latest` would work here too.
```
λ npx eslint src/*

C:\Code\upstream-snapp-cli\src\postinstall.js
  0:0  error  Parsing error: Invalid ecmaVersion

✖ 1 problem (1 error, 0 warnings)
```